jiraworklog/src/App.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component, Suspense } from 'react';
+import { Route, Switch } from 'react-router-dom';
+import Login from './Container/Login/Login';
+
+const Timesheet = React.lazy(() => import('./Container/Timesheet/Timesheet'));
+
+class App extends Component {
+  render() {
+    let route = (
```

Comment:
why we are still using let, it should be const right?
have you installed and configured eslint ? I am asking coz it should have automatically captured using eslint